src/main/java/training/supportbank/CsvReader.java

```diff
@@ -16,11 +16,9 @@
 
 public class CsvReader {
 
-    private ArrayList<Transaction> allTransactions = new ArrayList<>();
-    private ArrayList<User> allUsers = new ArrayList<>();
-    //test
-    public void test() throws IOException, ParseException {
-        CSVReader reader = new CSVReader(new FileReader("src/main/Transactions2014.csv"), ',','\'',1);
+
+    public void csvListGenerator(ArrayList<Transaction> allTransactions, ArrayList<User> allUsers,String fileName) throws IOException, ParseException {
+        CSVReader reader = new CSVReader(new FileReader(fileName), ',','\'',1);
```

Comment:
Nice job separating this out into its own method, makes the code a lot clearer to read

Also well done with using streams below, this is a prime example of when you would want to use them